src/tracing/config.rs

```diff
@@ -1,9 +1,4 @@
-use std::sync::Arc;
```

Comment:
To be checked but I think we can maybe just reuse the `ConfigureEvm` via [this](https://github.com/paradigmxyz/reth/blob/46c8255fa887dcc9d9b65aa9c1c47eeeabc4764d/crates/ethereum/evm/src/lib.rs#L30) since we don't need any special opcode for now.

Reply:
Yes, just fixed to use directly reth types